### DISCUSSION TITLE
Form Fields in Office Files Do Not Convert Properly into PDF

### DISCUSSION THREAD

**Discussion by roslove44** [gotenberg/gotenberg#976] [Q&A]
Hi!
I'm experiencing an issue with Gotenberg where the styles of form fields in Office documents are not preserved when converting to PDF. Specifically, fields such text inputs and select (option selected) do not retain their original formatting.

**Exemple :** 

> In Word

![image](https://github.com/user-attachments/assets/60b166e2-0812-4ead-a21b-c7b41c9fd345)

> -- PDF Output 

![image](https://github.com/user-attachments/assets/71b39ab2-6929-4694-91a4-7bd67588d6fb)

Could you provide guidance on how to ensure that form field styles and functionality are preserved during the conversion process? If there are specific settings or workarounds available, please advise on how to implement them.

Thank you for your assistance!

**Comment by gulien** [comment]
Hello @roslove44 👋 

Could you try with `exportFormFields=false`?

**Reply by roslove44** [reply]
Unfortunately I've already tried it, but always with the same result. It's weird, isn't it?

**Reply by gulien** [reply]
Yep, looks like LibreOffice does use its own "style" in such a scenario 🫤
